src/2025h1/all-hands.md

```diff
@@ -0,0 +1,92 @@
+- Have a two-day all-hands on May 16 and May 17, 2025.
+
+### The "shiny future" we are working towards
+
+The immediate goal is a very succesful and productive Rust All-Hands 2025.
+Hopefully, this will be the first step twoards the larger goal of having regular Rust All-Hands again.
+
+We should be able to use the feedback and lessons learned from this event for the next ones.
+Repeating an event tends to be much easier than organising one from scratch.
+
+## Design axioms
+
+- Accessibility. Ideally, everyone in the project should be able to attend the Rust All-Hands
+- Productivity and effectivity. We should optimally make use of the event, to make it worth everyone's time.
+- Low stress. The event or the planning of it should not burn anyone out. It is a tool to help the project and its members, after all!
+- Space for social events. The goal is not just to work on technical things together, but also to get to know each other and become a closer team.
+
+## Ownership and team asks
+
+**Owner:** Mara
+
+| Subgoal                                        | Owner(s) or team(s)            | Notes |
+| ---------------------------------------------- | ------------------------------ | ----- |
+| Decide to have another Rust All-Hands!         | ![Team][] [leadership-council] | Done! |
+| Pick the dates                                 | RustNL, leadership-council     | Done! |
+| Allocate funds for event                       | ![Team][] [leadership-council] | Done! |
+| Allocate funds for travel                      | Rust Foundation                | Done! |
+| Book the venue                                 | RustNL                         | Done! |
+| Catering / snacks / food / drinks              | RustNL                         | Done! |
+| [Register] for the Rust All-Hands 2025         | *every project member*         | Majority already signed up! |
+| Send out conformations/tickets                 | Mara                           |       |
+| Send out detailed information                  | Mara                           |       |
+| Answer logistical questions                    | Mara                           |       |
+| Interface between project and RustNL           | Mara                           | (Mara is part of both Rust and RustNL.) |
+| Make hotel reservations                        | RustNL                         | In progress |
+| Book hotel                                     | *all participants*             | RustNL will provide suggestions |
+| Book travel                                    | *all participants*             |       |
+| Team swag                                      |                                |       |
+| ↳ Decide on what to get (stickers? shirts?)    | ![Team][] [leadership-council] | Suggestions very welcome! |
+| ↳ Acquire it                                   | RustNL                         |       |
+| Make plans for what to do at the all-hands     | *every team*                   |       |
```

Comment:
The key challenge here is going to be scheduling meetings for teams with overlapping membership. Likely it'll probably make sense to stagger team meetings such that there are blocks of time with very few overlaps of "official" meetings (with the acknowledgement that maybe that means during those times the parallelism goes down).

Reply:
Yeah that's part of the reason for a "Planning team" to "Coordinate the overall process". Hopefully we can make the draft agendas public early so teams can work together directly to align their plans a bit. It's going to be a fun puzzle though. ^^

Reply:
Let me know if you want help with this. Happy to help coordinate with teams to figure out what they want/need.

Reply:
Welcome to the planning team. Feel free to invite others ;)